pom.xml

```diff
@@ -148,6 +148,12 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>script-security</artifactId>
```

Comment:
Probably better put in `dependencyManagement`, since it is not a direct dep AFAIK, but OK.